data-client/src/main/java/org/wikipedia/dataclient/mwapi/MwQueryPage.java

```diff
@@ -141,6 +151,29 @@ public void appendTitleFragment(@Nullable String fragment) {
         title += "#" + fragment;
     }
 
+    public boolean checkWhetherFileIsUsedInWikis() {
+        if (globalUsages != null && globalUsages.size() > 0) {
+            return true;
+        }
+
+        if (fileUsages == null || fileUsages.size() == 0) {
+            return false;
+        }
+
```

Comment:
How about just this?
```
for (final FileUsage fileUsage : fileUsages) {
    if ( ! fileUsage.title().contains("User:Didym/Mobile upload")) {
        return true;
    }
}
```

Reply:
Yes, this will work cause as soon as we detect usage other than ```User:Didym/Mobile Upload```, we return that it is in use. 

Will update, Thanks.